btcpay-dropbox/1.0.1/linuxamd64.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.7.2-alpine3.9
+
+WORKDIR /btcpay-dropbox
+
+COPY . /btcpay-dropbox
+
+RUN pip install dropbox
```

Comment:
Move this before the copy so that changing the dropbox-script will not invalidate the previous docker layers when you rebuild the image.